plugins/xkcd.py

```diff
@@ -22,7 +22,7 @@ def xkcd_info(xkcd_id, url=False):
 
 def xkcd_search(term):
     search_term = requests.utils.quote(term)
-    request = requests.get("http://www.ohnorobot.com/index.pl?s={}&Search=Search&"
+    request = requests.get("http://www.ohnorobot.com/index.php?s={}&Search=Search&"
```

Comment:
It may be better to use `http://www.ohnorobot.com/` instead of `http://www.ohnorobot.com/index.php` so it uses whatever page the site uses as the default.

Reply:
Good call, updated